robot_skills/src/robot_skills/hero.py

```diff
@@ -109,3 +106,40 @@ def move_to_hmi_pose(self):
         arm.send_joint_goal('arm_out_of_way', 0.0)
         self.base.force_drive(0, 0, rotation_speed, rotation_duration)
         arm.wait_for_motion_done()
+
+    def move_to_pregrasp_pose(self, arm, grasp_target):
+        """
+        This poses the robot for an inspect.
+
+        :param grasp_target: kdl.Frame with the pose of the entity to be grasp.
+        :return: boolean, false if something went wrong.
+        """
+        # calculate the arm_lift_link which must be sent
+        z_over = 0.4  # height the robot should look over the surface
+        z_hh = 0.9  # height of the robots head at z_arm=0
+        torso_to_arm_ratio = 2.0  # motion of arm/motion of torso
+        z_head = grasp_target.z() + z_over
+
+        if z_head < 1.3:
+            # we dont need to do stupid stuff
+            arm.send_joint_trajectory('prepare_grasp')
+            return True
+
+        # saturate the arm lift goal
+        z_arm = (z_head - z_hh) * torso_to_arm_ratio
+        z_arm = min(0.69, max(z_arm, 0.0))
+
+        pose = arm._arm.default_trajectories['prepare_grasp']
```

Comment:
This is not the official way of doing it! Use the public arm!

Reply:
Not possible, we cannot get the values of the default trajectories from the publicArm. And we shouldn't because normally you would not want to use them for anything.